src/actions/transitionProperties.ts

```diff
@@ -68,8 +68,8 @@ export const showTransitionProperties = (component: string, stateMachine: string
                         messageType,
                         jsonMessageString,
                         id: firstId,
-                        privateTopic: (privateTopic) ? privateTopic : session.getDefaultPrivateTopic(),
-                        privateTopics: session.getPrivateTopics()
+                        privateTopic: (privateTopic) ? privateTopic : session.privateTopics.getdefaultPublisherTopic(),
+                        privateTopics: session.privateTopics.getSubscriberTopics()
```

Comment:
why do you have `privateTopic` and `privateTopics` on the same structure?

Reply:
A relic from the reactiveApi, will be removed in next PR